test/models/user_test.rb

```diff
@@ -1,7 +1,37 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+   test "validity" do
+     user = User.new(name: "ryan", screen_name: "rasensio1")
+     assert user.valid?
+
+     user.name = nil
+     refute user.valid?
+
+     user.name = "ryan"
+     user.screen_name = nil
+     refute user.valid?
+   end
+
+   test "creates a valid user" do
+      mock_auth = OmniAuth.config.mock_auth[:github] = OmniAuth::AuthHash.new({
+        provider: 'twitter',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.